#! /bin/sh /usr/share/dpatch/dpatch-run
## autoconf_h323.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: make autoconf properly detect pwlib and openh323 from debs.
## DP: Upstream bug: http://bugs.digium.com/view.php?id=8699 .

@DPATCH@
diff -urNad asterisk-1.4.0~dfsg~/configure.ac asterisk-1.4.0~dfsg/configure.ac
--- asterisk-1.4.0~dfsg~/configure.ac	2006-12-18 18:55:46.000000000 +0200
+++ asterisk-1.4.0~dfsg/configure.ac	2006-12-31 02:59:45.000000000 +0200
@@ -719,7 +719,7 @@
 	if test "${HAS_PWLIB:-unset}" != "unset"; then
 		AST_CHECK_OPENH323_PLATFORM()
 
-		PLATFORM_PWLIB="pt_${PWLIB_PLATFORM}_r"
+		PLATFORM_PWLIB="pt"
 
 		AST_CHECK_PWLIB_BUILD([PWLib], [PWLIB],
 			[Define if your system has the PWLib libraries.],
@@ -743,7 +743,7 @@
 	AST_CHECK_OPENH323()
 	AST_CHECK_PWLIB_VERSION([OpenH323], [OPENH323], [openh323buildopts.h], [1], [17], [3])
 	AST_CHECK_OPENH323_BUILD()
-	PLATFORM_OPENH323="h323_${PWLIB_PLATFORM}_${OPENH323_SUFFIX}"
+	PLATFORM_OPENH323="openh323"
 	AST_CHECK_PWLIB_BUILD([OpenH323], [OPENH323],
 		[Define if your system has the OpenH323 libraries.],
 		[#include "ptlib.h"
